app/models/story_parser.rb

```diff
@@ -456,6 +457,10 @@ def download_from_lj(location)
     def download_chaptered_from_ffnet(location)
       raise Error, "Sorry, Fanfiction.net does not allow imports from their site."
     end
+
+    def download_chaptered_from_quotev(location)
```

Comment:
Unused method argument - `location`. If it's necessary, use `_` or `_location` as an argument name to indicate that it won't be used. You can also write as `download_chaptered_from_quotev(*)` if you want the method to accept any arguments but don't care about them.<br>Inconsistent indentation detected.